### ISSUE TITLE
on postgres .onConflict().ignore() should be able to take no columns

### ISSUE THREAD

**Issue by calvinmetcalf** [knex/knex#4552]
# Environment

Knex version:0.95.4
Database + version: postgres 12.7
OS: OSX

# Bug
running
```js
knex('table').insert({something:1}).onConflict().ignore();
```

generates

```sql
insert into "table" (something) values($1) on conflict ("true") do nothing;
```

when it should generate

```
insert into "table" (something) values($1) on conflict do nothing;
```

it looks like a fairly simply fix [around here](https://github.com/knex/knex/blob/228b49f4afb7d2cb03f4219b6cb0d1bb854d60f8/lib/dialects/postgres/query/pg-querycompiler.js#L79) which I might be able to open a pr for next week, amusing my wife doesn't give birth this weekend.

**Comment by calvinmetcalf** [comment]
update: she hasn't so I opened up a pull #4557

**Comment by kibertoad** [comment]
Released in 0.95.7
